index.html

```diff
@@ -4,33 +4,67 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="styles" href="css/styles.css" />
+    <link rel="stylesheet" href="/css/styles.css" />
   </head>
   <body>
     <header>
       <h1>Quiz App</h1>
     </header>
     <!-- I will use question-card as a class later to style it - What is better to use - or _ when I write a class?  -->
     <section class="question-card">
-      <h2>What is Riesling?</h2>
-      <button aria-label="Klick here to get answere">Show Answer</button>
-      <p>A Grape variety, that was esay</p>
-      <button>Hide Answer</button>
+      <div class="div_question-card">
```

Comment:
a card is a perfect example to use an `article` or a `section`.
its more sematic than a `div`